eureka/registry.go

```diff
@@ -135,7 +135,8 @@ func cfAppName(cfApps []plugin_models.GetAppsModel, cfAppGuid string) (string, e
 
 // Utility for operating on an application instance in the service registry
 
-type InstanceOperation func(accessToken string, eurekaUrl string, eurekaAppName string, instanceId string) error
+type InstanceOperation func(authClient httpclient.AuthenticatedClient, eurekaUrl string, eurekaAppName string, instanceId string, accessToken string) error
+//type InstanceOperation func(accessToken string, eurekaUrl string, eurekaAppName string, instanceId string) error
```

Comment:
Probably meant to remove this commented out line?

Reply:
@georgeharley Yes. Deleted, thanks.